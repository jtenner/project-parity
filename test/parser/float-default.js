const { OptionType } = require("../../lib");

module.exports = {
  argv: [],
  config: {
    optionA: {
      type: OptionType.Float,
      default: 42,
    },
  },
};
