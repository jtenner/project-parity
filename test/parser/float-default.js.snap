{
  args: [],
  cliProvided: Set(0) {},
  configProvided: Set(0) {},
  errors: [],
  options: { optionA: 42 },
  unknown: []
}
